Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -851,14 +851,9 @@
   - ANSWERING REQUESTS NOT EXPRESSED IN IAMBIC PENTAMETER
   - ARSON
   - ASKING FOR THINGS
-  - BEING CANADIAN
```

Comment:
This is on me, I should have swept this section more thoroughly when I did the cleanup. (#26703). Like with my PR, think of some new entries, so there's an equivalent variety.

E.g:

`BEING AN ENGINEER
BEING A MEDIC
BEING A CARGONAIN
BEING A SCIENTIST
BEING A PASSENGER`